include/flyft/spherical_mesh.h

```diff
@@ -8,11 +8,16 @@ namespace flyft
 class SphericalMesh : public Mesh
     {
     public:
-        SphericalMesh(int shape, double step,double origin);
+        SphericalMesh(double L, int shape);
+        SphericalMesh(double L, int shape, double origin);  
```

Comment:
Use R rather than L, since this is a sphere?
```suggestion
        SphericalMesh(double R, int shape);
        SphericalMesh(double R, int shape, double origin);  
```